src/main/blackbox/blackbox.h

```diff
@@ -23,7 +23,10 @@ void blackboxLogEvent(FlightLogEvent event, flightLogEventData_t *data);
 
 void initBlackbox(void);
 void handleBlackbox(void);
+void startInTestMode(void);
```

Comment:
The definitions in header file are actually not needed for all of these functions